doc/material/index.rst

```diff
@@ -63,10 +73,10 @@ Use ``mix-models materials-ix {SSP} build`` to add the material implementation o
 
     mix-models \
         --url="ixmp://ixmp_dev/MESSAGEix-GLOBIOM 1.1-R12/baseline_DEFAULT#21" \
-        --local-data "./data" material-ix SSP2 build --tag test
+        --local-data "./data" material-ix SSP2 build --tag test --nodes R12
 
-The output scenario name will be baseline_DEFAULT_test. An additional tag `--tag` can be used to add an additional suffix to the new scenario name.
-The mode option `--mode` has two different inputs 'by_url' (by default) or 'by_copy'.
+The output scenario name will be baseline_DEFAULT_test. An additional tag ``--tag`` can be used to add an additional suffix to the new scenario name.
+The mode option ``--mode`` has two different inputs 'by_url' (by default) or 'by_copy'.
```

Comment:
Have you checked the automatically built documentation pages that this looks correct? Two line below, a similar parameter is only given with one `, so please make sure it looks the same.

Reply:
@glatterf42 @macflo8 in general, for CLI commands and options we should use:
```rst
:program:`--foo=bar`
```
…using the Sphinx role provided for this purpose. This then gets styled in boldface.

However, please don't add this to TODOs for the current PR; simply keep in mind to update in the future.